### PR TITLE
Add usage documentation

### PR DESCRIPTION
**Pull request by chrissimpkins**
TODO:

- [ ] update build documentation link https://github.com/source-foundry/alt-hack/commit/11d96c59acfb92392cd763696c2ddb8107c7d222#diff-04c6e90faac2675aa89e2176d2eec7d8R22 when Hack v3.000 is merged to master
- [ ] update build documentation link https://github.com/source-foundry/alt-hack/pull/7/commits/11d96c59acfb92392cd763696c2ddb8107c7d222#diff-04c6e90faac2675aa89e2176d2eec7d8R47 when Hack v3.000 is merged to master

### CODE REVIEW

File: README.md
Comment (mynetx) [inline README.md]: Add trailing periods to these items.

File: README.md
Comment (mynetx) [inline README.md]: A link is missing?

File: README.md
Comment (mynetx) [inline README.md]: TTF or `*.ttf`.

File: README.md
Comment (mynetx) [inline README.md]: Period?

**Review by mynetx — CHANGES_REQUESTED** [review]

**Comment by chrissimpkins** [comment]
Want to add these changes? You have write access.  Feel free to write to dev branch so reflected in the PR.  We can discuss as changes are made.

**Comment by mynetx** [comment]
@chrissimpkins Yes course. Just not right now, either later today or tomorrow.

**Comment by chrissimpkins** [comment]
@jublo no rush.  We won't merge until we merge Hack dev to master (~Thursday).  Will be working on some doc updates after the Hack merge and if you haven't had a chance by then I will make all of your suggested edits and merge this.  If you get to it before I do feel free to merge to master once v3.000 release of Hack is in master.   These docs assume those changes.

**Pull request merged**
